polaris.shopify.com/pages/examples/data-table-with-fixed-first-columns.tsx

```diff
@@ -276,8 +276,8 @@ function DataTableWithFixedFirstColumnsExample() {
     ],
   ];
   const [sortedRows, setSortedRows] = useState(rows);
-  const showFixedColumns = useMedia('screen and (max-width: 850px)');
-  const fixedFirstColumns = showFixedColumns ? 2 : 0;
+  const {mdDown, mdOnly} = useBreakpoints();
+  const fixedFirstColumns = mdDown || mdOnly ? 2 : 0;
```

Comment:
**thought:* Should we use `lgDown` instead?
https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#media-query-variables

Reply:
yeah this is better, no issues with the example on 🎩 
@nickpresta  if you folks are happy with this, I'll merge 👍